shimmingtoolbox/unwrap/prelude.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" Description
+
+"""
+
+
+import numpy as np
+import os
+import nibabel as nib
+
+
+def prelude(complexArray, affine, mask=np.array([-1]), path2UnwrappedPhase="./unwrappedPhase.nii", isUnwrappingIn2D=True, isSavingNiftis=False):
+    """
+
+    Args:
+        complexArray:
+        affine:
+        mask:
+        path2UnwrappedPhase:
+        isUnwrappingIn2D:
+        isSavingNiftis:
+
+    Returns:
+
+    """
+    # Get absolute path
+    absPath = os.path.abspath(path2UnwrappedPhase)
+    dataSaveDirectory = os.path.dirname(absPath)
+
+    # Save complex image
+    # TODO: Save voxelSize when saving nifti (Maybe affine does it) (if it does, might be easier to just pass entire nibabel object)
+    phaseNii = nib.Nifti1Image(np.angle(complexArray[:-1, :-1, 0, 0]), affine)
+    magNii = nib.Nifti1Image(np.abs(complexArray[:-1, :-1, 0, 0]), affine)
+    nib.save(phaseNii, os.path.join(dataSaveDirectory, "rawPhase.nii"))
+    nib.save(magNii, os.path.join(dataSaveDirectory, "mag.nii"))
+
+    # Fill options
+    options = " "
+
+    if isUnwrappingIn2D:
+        options = options + "-s "
```

Comment:
You could use the += operator in Python. It is optimized for this kind of operation.